web/src/message_events.ts

```diff
@@ -375,6 +377,16 @@ export function insert_new_messages(
     return messages;
 }
 
+function topic_resolve_toggled(new_topic: string, original_topic: string): boolean {
```

Comment:
This isn't as robust as what I did on the server side for checking for long topic truncation when it was resolved, but it is what we currently have for filtering out topic resolve/unresolve entries when we analyze the message edit history array for the EDITED/MOVED indicators in the web app.